airgun/entities/contentview_new.py

```diff
@@ -23,25 +24,91 @@ def search(self, value):
         view = self.navigate_to(self, 'All')
         return view.search(value)
 
+    def publish(self, entity_name, values=None):
+        """Publishes to create new version of CV and promotes the contents to
+        'Library' environment.
+        :return: dict with new content view version table row; contains keys
+            like 'Version', 'Status', 'Environments' etc.
+        """
+        view = self.navigate_to(self, 'Publish', entity_name=entity_name)
+        if values:
+            view.fill(values)
+        view.next.click()
+        view.finish.click()
+        view.progressbar.wait_for_result()
+        view = self.navigate_to(self, 'Edit', entity_name=entity_name)
+        return view.versions.table.read()
+
+    def check_if_blank_in_french(self):
+        view = self.navigate_to(self, 'French')
+        return view.table.read()
 
 @navigator.register(NewContentViewEntity, 'All')
 class ShowAllContentViewsScreen(NavigateStep):
     """Navigate to All Content Views screen."""
 
-    VIEW = NewContentViewTableView
+    VIEW = ContentViewTableView
 
     @retry_navigation
     def step(self, *args, **kwargs):
         self.view.menu.select('Content', 'Lifecycle', 'Content Views')
 
 
+@navigator.register(NewContentViewEntity, 'French')
+class ShowAllContentViewsScreenFrench(NavigateStep):
+    """Navigate to All Content Views screen ( in French )"""
+
+    VIEW = ContentViewTableView
+
+    @retry_navigation
+    def step(self, *args, **kwargs):
+        self.view.menu.select('Contenu', 'Lifecycle', 'Content Views')
```

Comment:
how will you change the language of the browser to French is that part you are handling in the test? or any configuration? @jyejare are you okay with adding such scenarios in Robottelo?

Reply:
This part handles navigation AFTER the language has already been changed to french. Nothing more, nothing less. Right, Sam?

Reply:
Correct. This essentially models navigating to cv_ in a french language satellite _

In the test, I create a user and change their language to french.

Reply:
@sambible @omkarkhatavkar  As per the discussion with people leads / Sudhir , we as a functional QE are not supposed to either test/automate globalization BZs. 

@sambible Any reason why we have automated the BZ ?

Reply:
The BZ was about a whole empty page, right? That doesn't sound like a "globalization BZ".

Reply:
IT's not a globalization BZ, it's testing a specific customer Issue, where when we display certain characters in the UI, the pages will fail to display. This could happen in non-localization scenarios.

Reply:
@sambible @omkarkhatavkar Not sure whether such bugs comes into the category of Localization but I just wanted to ensure that we dont divert airgun to adapt something new for such a localization-specific case. Airgun is a pure functional testing tool. 

Also I really dont think we should cover such cases in regression testing thats less likely to reproduce in the future to catch the `regression` of it!